sonar-project.properties

```
@@ -0,0 +1,12 @@
+sonar.projectKey=geh-frontend-app
+sonar.organization=energinet-datahub
+
+# This is the name and version displayed in the SonarCloud UI.
+# sonar.projectName=DH3 Frontend App
+# sonar.projectVersion=1.0
+
+# Path is relative to the sonar-project.properties file. Replace "\" #by "/" on Windows.
+sonar.sources=./apps/dh/app-dh,./libs/dh
```

suggestion: Consider adding Watt and shared code
```suggestion
sonar.sources=./apps/dh/app-dh,./libs/dh,./libs/ui-watt,./libs/gf
```

---

I would rather have them as separate projects in sonarcloud. I assume that they have there own build workflow in GitHub - I might be wrong since this repo is new to me.

---

Watt itself (`libs/ui-watt`) is not currently independently buildable as its consumed directly by frontend applications in this monorepo. It does, however, have a deployment workflow for its living documentation (Storybook).

Shared libraries (`libs/gf`) are used by the DataHub frontend so it should be scanned by SonarCloud either here or elsewhere.